kani-driver/src/call_cargo.rs

```diff
@@ -80,10 +82,18 @@ impl KaniSession {
             });
         }
 
+        let mut symtabs = glob(&outdir.join("*.symtab.json"))?;
+        let mut metadata = glob(&outdir.join("*.kani-metadata.json"))?;
+        if std::env::var("IS_KANI_PROPTEST").is_ok() {
```

Comment:
Are we planning to enable this as a compilation flag or a runtime option?

Reply:
As of now, `scripts/cargo-kani` detects proptests automatically. I would be OK making it an argument, perhaps via `-Z`

Reply:
Yeah, it's probably better if the user has to opt-in at this point.